src/common/codeUtil.ts

```diff
@@ -8,12 +8,13 @@ export class CodeUtil {
             return '';
         }
         return original
-            .replace(/;.+/, '')
             .replace(/".*?"/g, '""') // replace string literals with empty string literal
             .replace(/{.*}/g, '') // remove matching braces
             .replace(/ +/g, ' ')
             .replace(/\bgui\b.*/gi, '')
-            .replace(/\b(msgbox)\b.+?%/gi, '$1');
+            .replace(/\b(msgbox)\b.+?%/gi, '$1')
```

Comment:
Let's add some unit tests for `purify` so we know what exactly we fixed and we don't break it in the future :)